### ISSUE TITLE
`bikeshed --version` outputs `???`

### ISSUE THREAD

**Issue by bramus** [speced/bikeshed#2487]
I have `3.11.13` installed. 

`bikeshed --version` outputs `???`. Looking at the code that happens when it can’t find `semver.txt`

https://github.com/speced/bikeshed/blob/9d285b3843d443ec93defdcd4147274035ff5d59/bikeshed/cli.py#L15-L21

This is weird, because the build manifest marks that file for inclusion, along with some other ones:

https://github.com/speced/bikeshed/blob/9d285b3843d443ec93defdcd4147274035ff5d59/MANIFEST.in#L1-L4

All 4 of those files don’t seem to be included in the published version:

```
ls -Al /opt/homebrew/lib/python3.10/site-packages/bikeshed
total 1216
-rw-r--r--   1 bramus  admin   9486 Mar  3 11:32 InputSource.py
-rw-r--r--   1 bramus  admin   1591 Mar  3 11:32 Spec-autolinks.css
-rw-r--r--   1 bramus  admin   2726 Mar  3 11:32 Spec-colors.css
-rw-r--r--   1 bramus  admin    555 Mar  3 11:32 Spec-counters.css
-rw-r--r--   1 bramus  admin   3939 Mar  3 11:32 Spec-darkmode.css
-rw-r--r--   1 bramus  admin    507 Mar  3 11:32 Spec-domintro.css
-rw-r--r--   1 bramus  admin    153 Mar  3 11:32 Spec-issues.css
-rw-r--r--   1 bramus  admin    204 Mar  3 11:32 Spec-mdlists.css
-rw-r--r--   1 bramus  admin   1354 Mar  3 11:32 Spec-position-annos.js
-rw-r--r--   1 bramus  admin   1253 Mar  3 11:32 Spec-selflinks.css
-rw-r--r--   1 bramus  admin  21804 Mar  3 11:32 Spec.py
-rw-r--r--   1 bramus  admin   2379 Mar  3 11:32 __init__.py
-rw-r--r--   1 bramus  admin     30 Mar  3 11:32 __main__.py
drwxr-xr-x  37 bramus  admin   1184 Mar  3 11:32 __pycache__
-rw-r--r--   1 bramus  admin  18105 Mar  3 11:32 biblio.py
-rw-r--r--   1 bramus  admin   6696 Mar  3 11:32 bigblocks.bsfont
drwxr-xr-x   3 bramus  admin     96 Mar  3 11:32 boilerplate
-rw-r--r--   1 bramus  admin  45815 Mar  3 11:32 boilerplate.py
drwxr-xr-x   7 bramus  admin    224 Mar  3 11:32 caniuse
-rw-r--r--   1 bramus  admin  26509 Mar  3 11:32 cli.py
-rw-r--r--   1 bramus  admin   3886 Mar  3 11:32 conditional.py
drwxr-xr-x   9 bramus  admin    288 Mar  3 11:32 config
-rw-r--r--   1 bramus  admin    874 Mar  3 11:32 constants.py
-rw-r--r--   1 bramus  admin  45204 Mar  3 11:32 datablocks.py
-rw-r--r--   1 bramus  admin    126 Mar  3 11:32 dfnpanels-dark.css
-rw-r--r--   1 bramus  admin   1116 Mar  3 11:32 dfnpanels.css
-rw-r--r--   1 bramus  admin   3934 Mar  3 11:32 dfnpanels.js
-rw-r--r--   1 bramus  admin   6803 Mar  3 11:32 dfnpanels.py
drwxr-xr-x   5 bramus  admin    160 Mar  3 11:32 dfns
-rw-r--r--   1 bramus  admin    232 Mar  3 11:32 extensions.py
-rw-r--r--   1 bramus  admin   2844 Mar  3 11:32 fingerprinting.py
-rw-r--r--   1 bramus  admin   8519 Mar  3 11:32 fonts.py
-rw-r--r--   1 bramus  admin    368 Mar  3 11:32 func.py
drwxr-xr-x   8 bramus  admin    256 Mar  3 11:32 h
-rw-r--r--   1 bramus  admin   5028 Mar  3 11:32 headings.py
-rw-r--r--   1 bramus  admin  27602 Mar  3 11:32 highlight.py
-rw-r--r--   1 bramus  admin  23978 Mar  3 11:32 idl.py
-rw-r--r--   1 bramus  admin   7495 Mar  3 11:32 includes.py
drwxr-xr-x   4 bramus  admin    128 Mar  3 11:32 inlineTags
-rw-r--r--   1 bramus  admin  10841 Mar  3 11:32 issuelist.py
-rw-r--r--   1 bramus  admin    185 Mar  3 11:32 language.py
-rw-r--r--   1 bramus  admin   1582 Mar  3 11:32 lexers.py
-rw-r--r--   1 bramus  admin    397 Mar  3 11:32 line.py
drwxr-xr-x  10 bramus  admin    320 Mar  3 11:32 lint
drwxr-xr-x   6 bramus  admin    192 Mar  3 11:32 markdown
drwxr-xr-x   7 bramus  admin    224 Mar  3 11:32 mdn
-rw-r--r--   1 bramus  admin   6871 Mar  3 11:32 messages.py
-rw-r--r--   1 bramus  admin  58538 Mar  3 11:32 metadata.py
-rw-r--r--   1 bramus  admin   2598 Mar  3 11:32 publish.py
-rw-r--r--   1 bramus  admin    973 Mar  3 11:32 pyg.py
-rw-r--r--   1 bramus  admin  47208 Mar  3 11:32 railroaddiagrams.py
-rw-r--r--   1 bramus  admin   8540 Mar  3 11:32 railroadparser.py
drwxr-xr-x  10 bramus  admin    320 Mar  3 11:32 refs
-rw-r--r--   1 bramus  admin   1292 Mar  3 11:32 repository.py
-rw-r--r--   1 bramus  admin   6739 Mar  3 11:32 retrieve.py
drwxr-xr-x   5 bramus  admin    160 Mar  3 11:32 shorthands
drwxr-xr-x  24 bramus  admin    768 Mar  3 11:32 spec-data
drwxr-xr-x   5 bramus  admin    160 Mar  3 11:32 stringEnum
-rw-r--r--   1 bramus  admin   1941 Mar  3 11:32 t.py
-rw-r--r--   1 bramus  admin   5304 Mar  3 11:32 test.py
-rw-r--r--   1 bramus  admin    527 Mar  3 11:32 testsuite.py
drwxr-xr-x   5 bramus  admin    160 Mar  3 11:32 translate
-rw-r--r--   1 bramus  admin  69474 Mar  3 11:32 unsortedJunk.py
drwxr-xr-x  15 bramus  admin    480 Mar  3 11:32 update
drwxr-xr-x   6 bramus  admin    192 Mar  3 11:32 wpt
```

When I build locally (`python setup.py sdist bdist_wheel`) and extract the archive, I see the files are included

```
bramus in ~/repos/prs/bikeshed/dist/bikeshed-3.11.13 on main λ ls -Al
total 112
-rw-r--r--   1 bramus  primarygroup   7.4K Mar  3 11:38 Bikeshed.tmLanguage
-rw-r--r--   1 bramus  primarygroup   6.9K Mar  3 11:38 LICENSE
-rw-r--r--   1 bramus  primarygroup   173B Mar  3 11:38 MANIFEST.in
-rw-r--r--   1 bramus  primarygroup   5.5K Mar  3 12:12 PKG-INFO
-rw-r--r--   1 bramus  primarygroup   5.0K Mar  3 11:38 README.md
drwxr-xr-x  65 bramus  primarygroup   2.0K Mar  3 12:12 bikeshed
drwxr-xr-x   8 bramus  primarygroup   256B Mar  3 12:12 bikeshed.egg-info
drwxr-xr-x   7 bramus  primarygroup   224B Mar  3 12:12 docs
-rw-r--r--   1 bramus  primarygroup    31B Mar  3 11:38 pyproject.toml
-rw-r--r--   1 bramus  primarygroup   1.1K Mar  3 11:38 requirements.txt
-rw-r--r--   1 bramus  primarygroup     7B Mar  3 11:38 semver.txt
-rw-r--r--   1 bramus  primarygroup   2.3K Mar  3 12:12 setup.cfg
-rw-r--r--   1 bramus  primarygroup   1.1K Mar  3 11:38 setup.py
```

However, only the `bikeshed` folder ends up being installed into `/opt/homebrew/lib/python3.10/site-packages/` in the end.

I’m unfamiliar with the structure of Python Eggs, but I think the `top_level.txt` has to do something with this:

```
bramus in ~/repos/prs/bikeshed/dist/bikeshed-3.11.13 on main λ cat bikeshed.egg-info/top_level.txt 
bikeshed
```

Easiest solution here seems to be to simply copy `semver.txt` into the built `bikeshed` folder before packaging things up.

**Comment by tabatkins** [comment]
> However, only the bikeshed folder ends up being installed into /opt/homebrew/lib/python3.10/site-packages/ in the end.

Ohhhhh, *that's* why this happens! I've been puzzling over this issue for a long time, actually.

Yeah, this should be a super easy fix, then.

**Comment by tabatkins** [comment]
See if it works now for 3.11.14?

**Comment by bramus** [comment]
Verified fixed in 3.11.16

```
λ bikeshed --version
3.11.16
```
